v3/src/components/graph/adornments/movable-line/movable-line-model.ts

```diff
@@ -0,0 +1,69 @@
+import { Instance, types } from "mobx-state-tree"
+import { AdornmentModel, IAdornmentModel, IUpdateCategoriesOptions, PointModel,
+         kInfinitePoint } from "../adornment-models"
+import { Point } from "../../graphing-types"
+import { IAxisModel } from "../../../axis/models/axis-model"
+import { computeSlopeAndIntercept } from "../../utilities/graph-utils"
+import { kMovableLineType } from "./movable-line-types"
+
+export const MovableLineParams = types.model("MovableLineParams", {
```

Comment:
I'm thinking perhaps this would be better named something like `MovableLineInstance` -- what do you think?

Reply:
Yeah, that makes sense 👍